src/CoinbaseWalletSDK.ts

```diff
@@ -61,6 +61,7 @@ export class CoinbaseWalletSDK {
     if (!options.uiConstructor) {
       uiConstructor = opts => new WalletSDKUI(opts);
     } else {
+      /** @deprecated */
       uiConstructor = options.uiConstructor;
```

Comment:
Doesn't seem to referenced/used anywhere.

Reply:
I think the extension uses this.